src/journal.c

```diff
@@ -150,7 +150,7 @@ Journal *Journal_GetSavedPage(Journal *journal, BOOL param1)
             journal[v1] = journal[v1 - 1];
         }
 
-        Jounral_Init(&journal[0]);
+        Journal_Init(&journal[0]);
```

Comment:
```suggestion
        Journal_Init(journal);
```

Reply:
if you look up in the function, journal is an array, so it makes sense to write it like this